enhancements/update/cvo-log-level-api.md

```diff
@@ -0,0 +1,327 @@
+
+Same as standalone clusters.
+
+### Implementation Details/Notes/Constraints
+
+This enhancement proposes to create a new `operator/v1/types_clusterversion.go` 
+file in the [OpenShift API repository](https://github.com/openshift/api). 
+Meaning the new data types will be part of the [`github.com/openshift/api/operator/v1`][github.com/openshift/api/operator/v1]
+package. A new feature-gated cluster scoped configuration API resource will be 
+defined to configure the CVO.
+
+The types are inspired by the existing [`OperatorSpec`][OperatorSpec]
+and [`OperatorStatus`][OperatorStatus]
+structures and copy over some of the fields. Only the  
+`OperatorLogLevel` field is introduced to the `spec` field as there is currently
+no need to support the rest of the [`OperatorSpec`][OperatorSpec]
+fields or any new fields. A similar case applies to the 
+[`OperatorStatus`][OperatorStatus] fields. The `ObservedGeneration` field is 
+used to give feedback regarding the last generation change the CVO has dealt
+with. There is currently no need to support additional fields.
+
+The `ClusterVersionOperator` CRD will be behind a new 
+`ClusterVersionOperatorConfiguration` FeatureGate that will be used to 
+control the development of the CVO configuration.
+
+[github.com/openshift/api/operator/v1]: https://github.com/openshift/api/tree/master/operator/v1
+[OperatorSpec]: https://github.com/openshift/api/blob/f89ab92f1597eaed4de5b947c1781adde2bf42fb/operator/v1/types.go#L54
+[OperatorStatus]: https://github.com/openshift/api/blob/f89ab92f1597eaed4de5b947c1781adde2bf42fb/operator/v1/types.go#L112
+
+The proposed contents of the new `operator/v1/types_clusterversion.go` file:
+
+```go
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ClusterVersionOperator holds cluster-wide information about the Cluster Version Operator.
+//
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
+// +openshift:compatibility-gen:level=1
+// +openshift:file-pattern=cvoRunLevel=0000_00,operatorName=cluster-version-operator,operatorOrdering=01
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:path=clusterversionoperators,scope=Cluster
+// +openshift:api-approved.openshift.io=https://github.com/openshift/api/pull/2044
+// +openshift:enable:FeatureGate=ClusterVersionOperatorConfiguration
+type ClusterVersionOperator struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// metadata is the standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	metav1.ObjectMeta `json:"metadata"`
+
+	// spec is the specification of the desired behavior of the Cluster Version Operator.
+	// +kubebuilder:validation:Required
+	// +required
```

Comment:
Only one of these is required, for now prefer the kubebuilder version

Reply:
I have updated the enhancement, thanks.